test/repl/proxysql_repl_tests/exec_all_repl_tests.sh

```
@@ -0,0 +1,113 @@
+  source ../env.sh
+else
+  cd ${SCRIPTPATH}
+fi
+
+. constants
+
+# export env vars from .env to the environment of subsequent commands
+set -a
+. .env
+
+if [[ -z ${WORKSPACE} || ! -f ${WORKPACE}/src/proxysql ]]; then
```

Shellcheck (0.11.0)

[info] Possible misspelling: WORKPACE may not be assigned. Did you mean WORKSPACE?

(SC2153)